core/src/com/unciv/ui/newgamescreen/MapOptionsTable.kt

```diff
@@ -71,6 +71,10 @@ class MapOptionsTable(private val newGameScreen: NewGameScreen): Table() {
             newGameScreen.updateTables()
         }
 
+        // Pre-select custom if any map saved within last 15 minutes
```

Comment:
NICE QOL improvement!

Reply:
Thanks but - see comment `
        // This is a kludge - the better way would be to have a "play this map now" menu button in the editor
        // (which would ideally not even require a save file - which makes implementation non-trivial)` in there...